python/arcticdb/version_store/library.py

```diff
@@ -14,12 +14,12 @@
 from arcticdb.supported_types import Timestamp
 
 from arcticdb.version_store.processing import QueryBuilder
-from arcticdb.version_store._store import NativeVersionStore, VersionedItem
+from arcticdb.version_store._store import NativeVersionStore, VersionedItem, VersionQueryInput
 from arcticdb_ext.exceptions import ArcticException
 import pandas as pd
 import numpy as np
 import logging
-
+from arcticdb.log import version as log
```

Comment:
err, this looks wrong to me??

Reply:
oh yes, no longer necessary, since it was used by the warnings caused when we uses non-used fields from ReadRequest in batch read metadata